connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -0,0 +1,246 @@
+openapi: 3.0.0
+info:
+  description: |
+    Connector Builder Server API
+
+  version: "1.0.0"
+  title: Connector Builder Server API
+  contact:
+    email: contact@airbyte.io
+  license:
+    name: MIT
+    url: "https://opensource.org/licenses/MIT"
+externalDocs:
+  description: Find out more about Connector Builder
+  url: "https://docs.airbyte.com/connector-development/config-based/overview/"
+
+paths:
+  /v1/stream/read:
+    post:
```

Comment:
how come we want these endpoints defined as `POST` calls? Because we're using a request body? Or I guess a real sync is sort of a POST call since the end result has records written to a destination, even if the test one doesnt?

Reply:
Yeah I was sort of following the convention in our main Airbyte API where we use post calls and pass all of the inputs in the request body. Especially for this API, since we need to pass the entire connector definition contents to the server I felt like that needed to go in a request body as opposed to being something like a URL parameter. And we can't have a request bodies for get requests